app/src/main/java/com/github/sdpteam15/polyevents/fragments/ProfileFragment.kt

```diff
@@ -53,9 +52,13 @@ class ProfileFragment : Fragment() {
         userInfoLiveData.observe(this) { userInfo ->
             viewRoot.findViewById<EditText>(R.id.profileName).setText(userInfo!!.name)
```

Comment:
I think this should be userInfo!!.displayName instead of name based on the refactor you've done in the UserEntity class

Reply:
Yes this wasn't clear because the text input in the layout had as description "Full Name", which I thought refers to the name of the user. Are we sure it's the display name here that we want and not the name?

Reply:
We could store the displayName attribute of the Firesbase Authentication as the name of the user and remove the displayName attribute from the UserEntity so that we only have one attribute referring to the name

Reply:
Ok this should make things easier. So if we remove displayName, the "name that will be displayed" will be by default the user's name.